### DISCUSSION TITLE
Send Later keeps corrupting my drafts folder

### DISCUSSION THREAD

**Discussion by inashrub** [Extended-Thunder/send-later#431] [Q&A]
Hi,
I have a number of email set up to send later - once a year for each separate message.  I have about 30 queued at the moment, but hope to increase that to more than 100.  But...  I keep getting this message on startup...

> Send Later: Folder /INBOX/Drafts may be corrupt. Please open its properties and repair it. See https://extended-thunder.github.io/send-later/#corrupt-drafts-error. Specifically, it appears that message “Antivirus renewal” <43cf4eae-739c-dd25-3721-360eb8c9adfb@xxx.com> has been sent before, but still exists in the Drafts folder. This may or may not be indicative of a more serious problem. You might simply try deleting (or re-scheduling) the message in question. Send Later will skip this message, but continue processing your other scheduled draft messages when you close this alert.

I've followed the advice and have repaired the drafts folder, I've also saved a new message into the Drafts folder then deleted it, but the message keeps coming back.  The reference above means nothing to me, so I can't track down with of the messages is the problem.  I could delete them all and start again, but that defeats the purpose of send again.

Any ideas greatly appreciated!

TIA

**Comment by mbaierl** [comment]
Same issue here, every time I am sending a message my Sent and/or Draft folders are corrupt.
When I check the folder flags I see that the Sent folder has "Sent" and "Draft" activated. Removing the wrong flag helps... until I send the next message.

I've searched for the underlying Thunderbird bug but can't find anything...

Any idea what is causing this issue?

**Reply by Tom-2019** [reply]
Once you set up all draft folders in the local folder the problem should disappear.

For reoccurring messages I would try to turn off the virus scanner temporarily.

**Comment by jikamens** [comment]
If there's a message in your Drafts folder that's due to be sent when you start Thunderbird, and Send Later notices that and schedules it to be sent before Thunderbird connects to your IMAP server on startup, OR if Send Later schedules a message when Thunderbird is in offline mode, then when Thunderbird reconnects to your IMAP server, rather than telling the server that the message was deleted from the Drafts folder, it could resurrect the message. I think this is a TB bug, not a Send Later bug. Is it possible that this scenario is what you're seeing?

**Comment by jikamens** [comment]
Closing because no response. Please [open an issue](https://github.com/Extended-Thunder/send-later/issues/new) if you're still having this problem.
